open-api/index.html

```diff
@@ -0,0 +1,62 @@
+<!-- HTML for static distribution bundle build -->
```

Comment:
If we move this to `src/main/resources/public` it will be served as default page.